src/migrate/Migrator.js

```diff
@@ -419,15 +438,22 @@ export default class Migrator {
   }
 
   setConfig(config) {
-    return assign({}, CONFIG_DEFAULT, this.config || {}, config);
+    const newConfig = assign({}, CONFIG_DEFAULT, this.config || {}, config);
```

Comment:
Are you sure about naming? There is not much new about it, it is mostly a composite or combined one.

Reply:
It is going to be the new config was my thinking?